src/plugins/vue-i18n.js

```diff
@@ -18,7 +18,9 @@ async function loadLanguageAsync(i18n, lang) {
     i18n.loadedLanguages = [defaultLocale];
   }
   if (!i18n.loadedLanguages.includes(lang)) {
-    const msgs = await import(/* webpackChunkName: "lang-[request]" */ `@/locales/client/${lang}`);
+    // Fix ESLint error in template string
+    // eslint-disable-next-line prefer-template
+    const msgs = await import(/* webpackChunkName: "lang-[request]" */ '@/locales/client/' + lang);
```

Comment:
What would be the error if we indded use template string?

Reply:
```
TypeError: Cannot read property 'range' of null
Occurred while linting /Users/nwingt/Repos/likecoin-button/src/plugins/vue-i18n.js:22
    at SourceCode.getTokenBefore (/Users/nwingt/Repos/likecoin-button/src/node_modules/eslint/lib/token-store/index.js:303:18)
    at checkSpacingBefore (/Users/nwingt/Repos/likecoin-button/src/node_modules/eslint/lib/rules/template-curly-spacing.js:60:42)
    at TemplateElement (/Users/nwingt/Repos/likecoin-button/src/node_modules/eslint/lib/rules/template-curly-spacing.js:119:17)
    at /Users/nwingt/Repos/likecoin-button/src/node_modules/eslint/lib/util/safe-emitter.js:45:58
    at Array.forEach (<anonymous>)
    at Object.emit (/Users/nwingt/Repos/likecoin-button/src/node_modules/eslint/lib/util/safe-emitter.js:45:38)
    at NodeEventGenerator.applySelector (/Users/nwingt/Repos/likecoin-button/src/node_modules/eslint/lib/util/node-event-generator.js:251:26)
    at NodeEventGenerator.applySelectors (/Users/nwingt/Repos/likecoin-button/src/node_modules/eslint/lib/util/node-event-generator.js:280:22)
    at NodeEventGenerator.enterNode (/Users/nwingt/Repos/likecoin-button/src/node_modules/eslint/lib/util/node-event-generator.js:294:14)
    at CodePathAnalyzer.enterNode (/Users/nwingt/Repos/likecoin-button/src/node_modules/eslint/lib/code-path-analysis/code-path-analyzer.js:632:23)
```